File: chatwoot/python/src/DeleteAnAccountExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        api.AccountsApi(api_client).delete_an_account(
            account_id=None,
        )
    except ApiException as e:
        print("Exception when calling AccountsApi#delete_an_account: %s\n" % e)

File: chatwoot/ruby/src/DeleteAnAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    ChatwootClient::AccountsApi.new.delete_an_account(
        0, # account_id
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling AccountsApi#delete_an_account: #{e}"
end
